### PR TITLE
Izaak with prezto

### PR DESCRIPTION
**Pull request by karlhungus**
This is my shit switch to prezto

@pseudomuto fyi i did this -- and i just didn't like it... things like fasd are changed so they don't use z but j (which i did use), but none of the themes matched what i liked... i probably didn't check it enough to see if it was fast or not... anyway thought you might want to know.

### CODE REVIEW

**Comment by karlhungus** [comment]
Things i didn't like
- themes  - couldn't find one to replace [crunch](https://github.com/robbyrussell/oh-my-zsh/blob/d186bc30d071559a36bb190a3c3b329b882ec183/themes/crunch.zsh-theme)
- fasd with `j` access, beyond not working, i have to learn something new ... no thanks!

File: lib/deltas/source-repo.sh [hunk 1]
```diff
@@ -25,7 +27,14 @@ applied() {
 apply() {
   if [ ! -d "${target_dir}" ]; then
     mkdir -p "${target_dir}"
-    git clone "https://github.com/${repo}" "${target_dir}"
+    if [ $grecursive == "true" ]
+    then
+      echo "recursive ${repo} ${target_dir}"
```
Comment (karlhungus) [inline lib/deltas/source-repo.sh:32]: 😬

**Comment by karlhungus** [comment]
opinion

**Comment by pseudomuto** [comment]
Good enough for me...ohmyzsh remains! :smile:

**Pull request closed without merging**